image/image_linux.go

```diff
@@ -481,9 +481,16 @@ func (s *imageSeeder) deriveInfoForLocalSnaps(f seedwriter.SeedAssertionFetcher,
 		snaps[sn] = aRefs
 	}
 
-	// Check if there are remaining local components
+	// Check if there is a local components that did not belong to one of
+	// the local snaps
 	errMsg := ""
-	for path := range s.w.ByPathLocalComponents() {
+compsLoop:
+	for path, cinfo := range s.w.ByPathLocalComponents() {
```

Comment:
that seems something that Writer should do as soon as all the info is available instead

Reply:
Changed now by adding the check to w.InfoDerived